internal/service/employeeaccessgrant/resource.go

```diff
@@ -2,20 +2,27 @@ package employeeaccessgrant
 
 import (
 	"context"
+	"net/http"
 
 	"github.com/hashicorp/terraform-plugin-framework/resource"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 )
 
-const employeeAccessGrantName = "employee_access_grant"
+const (
+	resourceName     = "employee_access_grant"
```

Comment:
I think I like this more `generic` names. It may also simplify automation

Reply:
good point about generic names being more "automation friendly" cc @AgustinBettati